### ISSUE TITLE
Publish 1.0.1 package in nuget?

### ISSUE THREAD

**Issue by xumix** [DotNetAnalyzers/StyleCopAnalyzers#2342]
please

**Comment by stffabi** [comment]
Yes, this is currently a serious problem.

**Comment by mikes-gh** [comment]
I don't believe 1.0.1 will ever be released as there was a problem with it.
What did you need 1.0.1 for?
I am awaiting netcore/xplat-msbuild support as currently cant use the analyser in vscode.

**Comment by xumix** [comment]
@mikes-gh well, there were some fixes that I wanted and I'd like a fresh build to be available from official store

**Comment by mikes-gh** [comment]
I think the big blocker is netstandard support which doesn't appear to be straight forward.  Any news @sharwell

**Comment by mikes-gh** [comment]
@xumix If you were waiting for fixes I don't believe there was anything in 1.0.1 that wasn't in 1.0.0 
1.0.1 was an attempt at solving a problem with JSON.net on netstandard.

**Comment by xumix** [comment]
@mikes-gh oops, i'm sorry, I've waited for 1.1 fixes

**Comment by sharwell** [comment]
@xumix I'm going to treat this like the title says currently. If you meant a different number, feel free to open a different issue.

The 1.0.1 release was intended to fix some compatibility issues with .NET Core users. In practice, it did not behave as we intended so I unlisted the package in NuGet. I'm still working with others on the Roslyn team to come up with a good story for analyzers in cross-platform development scenarios. I don't have news yet but hopefully soon; it's causing a big pain point and not just for user of this project.
